SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK.h

```diff
@@ -0,0 +1,11 @@
+//
+
+#import <UIKit/UIKit.h>
```

Comment:
Are this empty lines needed?

Reply:
I don't know, but as these files are all autogenerated, I have no real control over them.